samples/test/parameter_values.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test pipeline for bug report https://github.com/kubeflow/pipelines/issues/5830"""
```

Comment:
I wonder if we may have some test sample that we can plug-in this test case instead of creating a whole new pipeline. 
I'm a little concern about the long-term maintainability if we might end up with a number of similar pipelines with minor differences. WDYT?

Reply:
These are good arguments!
I will try to find a sample to add these cases.

Reply:
Added these cases in existing samples